modify.md

```diff
@@ -54,7 +54,27 @@ class $modify(MenuLayer) {
 
 (or you could be questionable and do `this->MenuLayer::init()` but who am I to judge)
 
-This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency. 
+This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency.
+
+We can also improve this function by getting GD Window Size that can later be used to place stuff more precisely.
```

Comment:
"Window Size" should be all in lower case (and instead of 'GD' you should use 'the')

Reply:
The end of this sentence is also a bit wonky. In general, if a sentence has the word 'stuff' in it, that's indicative of poor writing. It's nearly always better to specify exactly what 'stuff' is being referred to. Also, getting the window size doesn't improve the function itself? The main point of getting it is to center stuff on screen, which is also 90% of the use cases for getting the window size, so citing that as motivation would be better to justify that line of code than to say it "improve(s) this function".

I.E. this would be better as something like "In order to center our label, we can get the window size from GD and move the label to be halfway across horizontally and vertically".

Reply:
Tried fixing [here](https://github.com/geode-sdk/docs/pull/5/commits/3d10d1bc9b6db67ffd8049bd65020cfed5678789)